src/main/java/appeng/fluids/helper/FluidCellConfig.java

```diff
@@ -100,7 +100,7 @@ public boolean isItemValidForSlot( int slot, ItemStack stack )
 		ItemStack is = Api.INSTANCE.definitions().items().dummyFluidItem().maybeStack( 1 ).get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.orod.org/coding_rules#rule_key=squid%3AS3655)